css/site.styl

```diff
@@ -108,11 +108,11 @@ $margin-basic = ($gutter * 2)
 $margin-mid = ($gutter * 3.5)
 $margin-large = ($gutter * 5)
 
-$font-size-basic = 15px
+$font-size-basic = 13px
 $font-size-larger = 1.3rem
-$font-size-h3 = golden-ratio(1rem , 1)
-$font-size-h2 = golden-ratio(1rem , 1.5)
-$font-size-h1 = golden-ratio(1rem , 2)
+$font-size-h3 = golden-ratio(1rem , .3)
```

Comment:
h3 has nothing to do with font-size semantically speaking.  its can be microscopic and still be semantically valid